pom.xml

```diff
@@ -77,7 +77,7 @@
         <dependency>
             <groupId>javax.xml.bind</groupId>
             <artifactId>jaxb-api</artifactId>
-            <version>2.3.1</version>
+            <version>2.4.0-b180830.0359</version>
```

Comment:
Is this correct release candidate version? It is very strange versioning number.

Reply:
You're right. Seems to be a pre-release

Reply:
```suggestion
            <version>2.3.1</version>
```